src/core_plugins/kibana/server/tutorials/apm/on_prem.js

```diff
@@ -41,6 +42,7 @@ import {
 export function onPremInstructions(apmIndexPattern) {
   const EDIT_CONFIG = createEditConfig();
   const START_SERVER_UNIX = createStartServerUnix();
```

Comment:
Is this now only used for OSX? Do you think it would make sense to rename as such then to avoid confusion?
Eg:
```suggestion
  const START_SERVER_OSX = createStartServerOSX();
```

Reply:
I was leaving open the possibility of adding a "linux" tab for those not using deb/rpm, in which case the instructions would be the same as osx.  I don't feel strongly about it.

Reply:
> I don't feel strongly about it

Me neither. Fine to leave it as-is.